app/Models/Form.php

```diff
@@ -12,4 +12,14 @@ class Form extends Model
     protected $guarded = [
         'id',
     ];
+
+    public function user()
+    {
+        return $this->belongsTo('App\User');
```

Comment:
```suggestion
        return $this->belongsTo(User::class);
```
fix tương tự các chỗ khác.